### PR TITLE
Updated config file doc to yaml 

### PR DESCRIPTION
**Pull request by MadelineMurray**
Updated config file doc to yaml

### CODE REVIEW

File: docs/Reference/CLI/CLI-Syntax.md [hunk 1]
```diff
@@ -174,7 +182,7 @@ TEKU_LOG_COLOUR_ENABLED=false
 ```
 
 ```bash tab="Configuration File"
-log-colour-enabled=false
+log-colour-enabled: False
```
Comment (macfarla) [inline docs/Reference/CLI/CLI-Syntax.md:185]: s/colour/color

File: docs/Reference/CLI/CLI-Syntax.md [hunk 2]
```diff
@@ -291,18 +299,18 @@ The default is `false`.
...
+TEKU_METRICS_CATEGORIES=BEACON,JVM,PROCESS
 ```
 
 ```bash tab="Configuration File"
-metrics-categories=["BEACONCHAIN", "JVM", "PROCESS"]
+metrics-categories: ["BEACON", "JVM", "PROCESS"]
 ```
 
-Categories for which to track metrics. Options are `BEACONCHAIN`, `JVM`, `PROCESS`, `NETWORK`.
+Categories for which to track metrics. Options are `BEACON`, `JVM`, `PROCESS`, `NETWORK`.
```
Comment (macfarla) [inline docs/Reference/CLI/CLI-Syntax.md:313]: LIBP2P is another option

File: docs/Reference/CLI/CLI-Syntax.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Reference/CLI/CLI-Syntax.md:185]: Fixed

File: docs/Reference/CLI/CLI-Syntax.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Reference/CLI/CLI-Syntax.md:313]: Added

**Pull request merged**